DataBaseTutorial/Migrations/DatabaseContextModelSnapshot.cs

```diff
@@ -0,0 +1,86 @@
+﻿// <auto-generated />
+using System;
+using DataBaseTutorial;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+
+namespace DataBaseTutorial.Migrations
+{
+    [DbContext(typeof(DatabaseContext))]
+    partial class DatabaseContextModelSnapshot : ModelSnapshot
+    {
+        protected override void BuildModel(ModelBuilder modelBuilder)
+        {
+#pragma warning disable 612, 618
+            modelBuilder
+                .HasAnnotation("ProductVersion", "3.1.4");
+
+            modelBuilder.Entity("DataBaseTutorial.Product", b =>
+                {
+                    b.Property<int>("ProductId")
+                        .ValueGeneratedOnAdd()
+                        .HasColumnType("INTEGER");
+
+                    b.Property<bool>("ProductAddled")
+                        .HasColumnType("INTEGER");
```

Comment:
No need to specify type explicitly. EF is capable of mapping CLR types to SQL types.
If field has no indexes or other constraints EF always matches it's name (if you don't need any different) and type. 
Simple example:
If you have public property 
`public int SomeProperty { get; set; }`
If you want to keep the same for database and use no other options and/or adjustments you just don't mention it in ModelBuilder pipeline. It will create column with name SomeProperty and SQL integer type.
So consider doing the same with other fields.